pkgs/applications/audio/boops/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, lib, fetchFromGitHub, xorg, cairo, lv2, libsndfile, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "BOops";
+  version = "1.4.0";
+
+  src = fetchFromGitHub {
+    owner = "sjaehn";
+    repo = pname;
```

Comment:
```suggestion
    repo = "BOops";
```

Reply:
Most of the other plugins in this series have the same capitalization:
https://github.com/NixOS/nixpkgs/blob/7ecb53ac5b2fa50eedb2f56310ce1a83ee6aa19e/pkgs/applications/audio/bslizr/default.nix#L4
https://github.com/NixOS/nixpkgs/blob/8284fc30c84ea47e63209d1a892aca1dfcd6bdf3/pkgs/applications/audio/bsequencer/default.nix#L4
https://github.com/NixOS/nixpkgs/blob/8284fc30c84ea47e63209d1a892aca1dfcd6bdf3/pkgs/applications/audio/bshapr/default.nix#L4
https://github.com/NixOS/nixpkgs/blob/8284fc30c84ea47e63209d1a892aca1dfcd6bdf3/pkgs/applications/audio/bjumblr/default.nix#L4

Reply:
Which is also not quite right because pname should match attr name in most cases.